src/map/mapreg_sql.c

```diff
@@ -105,6 +105,25 @@ static bool mapreg_setreg(int64 uid, int val)
 				SQL->EscapeStringLen(map->mysql_handle, tmp_str, name, strnlen(name, SCRIPT_VARNAME_LENGTH+1));
 				if( SQL_ERROR == SQL->Query(map->mysql_handle, "INSERT INTO `%s`(`varname`,`index`,`value`) VALUES ('%s','%u','%d')", mapreg->table, tmp_str, i, val) )
 					Sql_ShowDebug(map->mysql_handle);
+
+				struct SqlStmt *stmt = SQL->StmtMalloc(map->mysql_handle);
+
+				if (stmt == NULL) {
+					SqlStmt_ShowDebug(stmt);
+				} else {
+					const char *query = "INSERT INTO `%s` (`key`, `index`, `value`) VALUES (?, ?, ?)";
+					size_t len = strnlen(name, SCRIPT_VARNAME_LENGTH);
```

Comment:
better use ``safestrnlen``

Reply:
`strnlen()` isn't used anymore in later commits.